test/EFCore.PG.FunctionalTests/Query/PrecompiledQueryNpgsqlTest.cs

```diff
@@ -0,0 +1,2013 @@
+    {
+        await base.Same_captured_variable_twice_in_different_lambdas();
+
+        AssertSql(
+            """
+@__foo_0_startswith='X%'
+@__foo_0_endswith='%X'
+
+SELECT b."Id", b."Name", b."Json"
+FROM "Blogs" AS b
+WHERE b."Name" LIKE @__foo_0_startswith AND b."Name" LIKE @__foo_0_endswith
+""");
+    }
+
+    public override async Task Include_single()
+    {
+        await base.Include_single();
+
+        AssertSql(
+            """
+SELECT b."Id", b."Name", b."Json", p."Id", p."BlogId", p."Title"
+FROM "Blogs" AS b
+LEFT JOIN "Posts" AS p ON b."Id" = p."BlogId"
+WHERE b."Id" > 8
+ORDER BY b."Id" NULLS FIRST
+""");
+    }
+
+    public override async Task Include_split()
+    {
+        await base.Include_split();
+
+        AssertSql(
+    """
+SELECT b."Id", b."Name", b."Json"
+FROM "Blogs" AS b
+ORDER BY b."Id" NULLS FIRST
+""",
+    //
+    """
+SELECT p."Id", p."BlogId", p."Title", b."Id"
+FROM "Blogs" AS b
+INNER JOIN "Posts" AS p ON b."Id" = p."BlogId"
+ORDER BY b."Id" NULLS FIRST
+""");
+    }
+
+    public override async Task Final_GroupBy()
+    {
+        await base.Final_GroupBy();
+
+        AssertSql(
+            """
+SELECT b."Name", b."Id", b."Json"
+FROM "Blogs" AS b
+ORDER BY b."Name" NULLS FIRST
+""");
+    }
+
+    public override async Task Multiple_queries_with_captured_variables()
+    {
+        await base.Multiple_queries_with_captured_variables();
+
+        AssertSql(
+            """
+@__id1_0='8'
+@__id2_1='9'
+
+SELECT b."Id", b."Name", b."Json"
+FROM "Blogs" AS b
+WHERE b."Id" = @__id1_0 OR b."Id" = @__id2_1
+""",
+            //
+            """
+@__id1_0='8'
+
+SELECT b."Id", b."Name", b."Json"
+FROM "Blogs" AS b
+WHERE b."Id" = @__id1_0
+LIMIT 2
+""");
+    }
+
+    public override async Task Unsafe_accessor_gets_generated_once_for_multiple_queries()
+    {
+        await base.Unsafe_accessor_gets_generated_once_for_multiple_queries();
+
+        AssertSql(
+    """
+SELECT b."Id", b."Name", b."Json"
+FROM "Blogs" AS b
+""",
+    //
+    """
+SELECT b."Id", b."Name", b."Json"
+FROM "Blogs" AS b
+""");
+    }
+
+    [ConditionalFact]
+    public virtual void Check_all_tests_overridden()
+        => TestHelpers.AssertAllMethodsOverridden(GetType());
+
+    public class PrecompiledQueryNpgsqlFixture : PrecompiledQueryRelationalFixture
+    {
+        protected override ITestStoreFactory TestStoreFactory
+            => NpgsqlTestStoreFactory.Instance;
+
+        public override DbContextOptionsBuilder AddOptions(DbContextOptionsBuilder builder)
+        {
+            builder = base.AddOptions(builder);
+
+            // TODO: Figure out if there's a nice way to continue using the retrying strategy
+            var npgsqlOptionsBuilder = new NpgsqlDbContextOptionsBuilder(builder);
+            npgsqlOptionsBuilder.ExecutionStrategy(d => new NonRetryingExecutionStrategy(d));
+            return builder;
+        }
+
+        protected override async Task SeedAsync(PrecompiledQueryContext context)
```

Comment:
@ChrisJollyAU noticed after merging that this is still overridden - is that needed for some reason, or is the base implementation good here?

Reply:
The seeding does need to be overridden. EFCore.PG doesn't like DateTime when the `DateTimeKind` is `Unspecified`

Reply:
Makes sense 👍